src/content/developers/docs/scaling/validium/index.md

```diff
@@ -12,55 +12,136 @@ Validium is a [scaling solution](/developers/docs/scaling/) that enforces integr
+
+Funds belonging to validium users are controlled by a smart contract on Ethereum. Validiums offer near-instant withdrawals, much like ZK-rollups do; once the validity proof for a withdrawal request has been verified on Mainnet, users can withdraw funds by providing [Merkle proofs](/developers/tutorials/merkle-proofs-for-offline-data-integrity/). The Merkle proof validates the inclusion of the user’s withdrawal transaction in a verified transaction batch, allowing the on-chain contract to process the withdrawal. 
+
+However, we must also note that validium users can have their funds frozen and withdrawals restricted. This can happen if data availability managers on the validium chain withhold off-chain state data from users. Without access to transaction data, users cannot compute the Merkle proof required to prove ownership of funds and execute withdrawals.  
+
+This is the major difference between validiums and ZK-rollups—their positions on the data availability spectrum. Both solutions approach data storage differently, which has implications for security and trustlessness. 
+
+## How do validiums interact with Ethereum? {#how-do-validiums-interact-with-ethereum}
+
+Validiums are scaling protocols built on top of the existing Ethereum chain. Although it executes transactions off-chain, a validium chain is administered by a collection of smart contracts deployed on Mainnet including: 
+
+1. **Verifier contract**: The verifier contract verifies the validity of proofs submitted by the validium operator when making state updates. This includes validity proofs attesting to the correctness of off-chain transactions and data availability proofs verifying the existence of off-chain transaction data. 
+
+2. **Main contract**: The main contract stores state commitments (Merkle roots) submitted by block producers and updates the validium's state once a validity proof is verified on-chain. This contract also processes deposits to and withdrawals from the validium chain. 
+
+Validiums also rely on the main Ethereum chain for the following: 
+
+### Settlement {#settlement}
+
+Transactions executed on a validium cannot be fully confirmed until the parent chain verifies their validity. All business conducted on a validium must eventually be settled on Mainnet. The Ethereum blockchain also provides "settlement guarantees" for validium users, meaning off-chain transactions cannot be reversed or altered once committed to on-chain.
+
+### Security {#security}
+
+Ethereum, acting as a settlement layer, also guarantees the validity of state transitions on validium. Off-chain transactions executed on the validium chain are verified via a smart contract on the base Ethereum layer. 
+
+If the on-chain verifier contract deems the proof invalid, the transactions are rejected. This means operators must satisfy validity conditions enforced by the Ethereum protocol before updating the validium's state. 
+
 ## How does validium work? {#how-does-validium-work}
```

Comment:
same as in the state-channel PR - it feel strange to get this far into the content before hitting the "how do validiums work" header. In the original version "How do validiums work" was the top level header,which makes more sense to me.

Reply:
Left a comment on the state channel PR about my approach to the scaling pages, which is to provide a high-level overview, explain how the solution interacts with the Ethereum chain, and follow that up with a larger section detailing the low-level workflow of the scaling solution. 

The idea is that if a reader doesn't want to go deep into the details, they can read the high-level overview and the following section to understand the chain's relationship with Ethereum. Also, I should point out that "what is x?" is likely different from "how x works" in terms of the details both have--and not every reader necessarily wants the low-level details of how x works.

Reply:
yea ok, fair push-back.